swagger.yaml

```
@@ -528,7 +531,7 @@ paths:
         - "application/json"
       parameters:
         - in: path
-          name: name
+          name: id
           type: string
           required: true
           description: the group's name
```

Still referencing `name` rather than `id`:

```suggestion
          description: the group's ID
```